style.css

```diff
@@ -0,0 +1,115 @@
+body{background-color: pink;
+    display: grid inline grid;
+    font-family: 'Times New Roman', Times, serif
+}
+
+header{
+    display: inline;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 1;
+    grid-row-end: 4;
+    font-size: 16pt;
+    text-align: right;
+}
+a {
+    color: white;
+}
+.links {list-style: none;}
+.links :hover {color:blue;}
+.links:visited {color: gray}
+
+
+h1{
+    color:white;
+    text-align: Center;
+    font-family: italic;
+}
+
+
+.me{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+
+p {
+    color:black;
+    text-align: center;
+    font-size: 20pt;
+    grid-row-start: 2;
+    grid-row-end: 3;
+}
+
+.projects{
+    display: -webkit-flex;
+    -webkit-flex-wrap: wrap;
```

Comment:
For modern browsers we can get rid of these - they were only needed for older versions that aren't in popular use anymore.